back-end/resources/room_config.json

```diff
@@ -67,20 +67,25 @@
       "id": "telephone",
       "description": "The telephone can ring and display a message. It will also record the numbers turned, and send these as sequence",
       "input": {
-        "turningWheel": "num-array"
+        "turningWheel": "array"
       },
       "output": {
         "audio": "string",
         "ringTone": "string"
       }
     },
     {
-      "id": "door",
-      "description": "De deur kan open en dicht zijn en neemt geen input.",
+      "id": "testDevice",
```

Comment:
This needs to stay door, this is not a test file

Reply:
fixed this json